tools/run_tests/performance/build_performance.sh

```diff
@@ -43,6 +43,9 @@ do
   "go")
     tools/run_tests/performance/build_performance_go.sh
     ;;
+  *"php"*)
```

Comment:
nit: I'd just enumerate "php7" and "php7_protobuf_c" here, no need to use wildcards.
Also no reason to pass $CONFIG as an argument - that variable will be available in build_performance_php.sh anyway -  taking an env variable, passing it as a param only to make it an env variable seems unnecessary.

Reply:
Done. I use wildcards in case we need add benchmark for both php and php7. But that requires switch language version in VM, thus only focus on build benchmark for php7 now.